internal/libhive/testmanager.go

```diff
@@ -143,11 +143,10 @@ func (manager *TestManager) Terminate() error {
 		}
 		// ensure the db is updated with results
 		manager.doEndSuite(suiteID)
+		// remove the test suite's left-over docker networks.
+		manager.PruneNetworks(suiteID)
```

Comment:
This needs to move into `doEndSuite` so that it will also remove the networks when the suite ends in the normal way.